### PR TITLE
Skip liveness probe option

### PR DESCRIPTION
**Pull request by simonferquel**
On some Managed clusters (I look at you AKS !), https liveness probes fail with error 403.
To mitigate the issue temporary, this add a flag to disable liveness probes.

### CODE REVIEW

File: cmd/installer/main.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ func parseOptions(uninstall *bool, options *install.SafeOptions) *rest.Config {
 	flag.DurationVar(&options.OptionsCommon.ReconciliationInterval, "reconciliation-interval", constants.DefaultFullSyncInterval, "Interval of reconciliation loop")
 	flag.StringVar(&options.OptionsCommon.Tag, "tag", "latest", "Image tag")
 	flag.StringVar(&options.Etcd.Servers, "etcd-servers", "", "etcd server addresses")
+	flag.BoolVar(&options.SkipLivenessProbes, "skip-liveness-probes", false, "Don't specify a liveness probe on Controller and API server deployments. Can fix an issue in environements where HTTPS livenessprobe fail")
```
Comment (silvin-lubecki) [inline cmd/installer/main.go]: Can you rephrase the description and remove the `Don't` part but instead `Disable liveness probe ... Use this when HTTPS liveness probe fails.` ?

File: cmd/installer/main.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ func parseOptions(uninstall *bool, options *install.SafeOptions) *rest.Config {
 	flag.DurationVar(&options.OptionsCommon.ReconciliationInterval, "reconciliation-interval", constants.DefaultFullSyncInterval, "Interval of reconciliation loop")
 	flag.StringVar(&options.OptionsCommon.Tag, "tag", "latest", "Image tag")
 	flag.StringVar(&options.Etcd.Servers, "etcd-servers", "", "etcd server addresses")
+	flag.BoolVar(&options.SkipLivenessProbes, "skip-liveness-probes", false, "Disable liveness probe ... Use this when HTTPS liveness probe fails.")
```
Comment (chris-crone) [inline cmd/installer/main.go]: shouldn't have `...` or `.` at the end of the sentence

File: cmd/installer/main.go (hunk 1 above)
Comment (silvin-lubecki) [reply cmd/installer/main.go]: Yeah sorry, the ... mean fill it with the remaining part 😅

Comment (silvin-lubecki) [reply cmd/installer/main.go]: Disable liveness probe on Controller and API server deployments. Use this when HTTPS liveness probe fails.

**Comment by simonferquel** [comment]
@silvin-lubecki PTAL

**Comment by chris-crone** [comment]
@simonferquel Does this mean we can remove the liveness probe note on this? https://github.com/docker/compose-on-kubernetes/blob/master/docs/install-on-aks.md
